### PR TITLE
PR for UNIX-class platform PLIC spec.

### PR DESCRIPTION
**Pull request by changab**
PLIC spec is revised according to UNIX-class platform meeting.
- Remove PLIC flexibility from this spec. Only mention standard PLIC which compliant with SiFive PLIC design.
- Add images of figures mentioned in this spec.
- Separate PLIC ACPI spec from this spec to riscv-plic-acpi.adoc.

### CODE REVIEW

File: riscv-plic.adoc [hunk 1]
```diff
@@ -17,102 +18,291 @@ https://creativecommons.org/licenses/by/4.0/.
 This document contains the RISC-V platform-level interrupt controller (PLIC)
 specification, which defines an interrupt controller specifically designed to
 work in the context of RISC-V systems.  The PLIC multiplexes various device
-interrupts onto the external interrupt lines of various hart contexts, with
-hardware support for interrupt priorities.
+interrupts onto the external interrupt lines of various Hart contexts, with
+hardware support for interrupt priorities. +
+This specification defines the general PLIC architecture and operation parameters.
+The PLIC which claimed as PLIC-Compliant standard PLIC should follow the
+implementations mentioned in below sections.
```
Comment (Kevin-Andes) [inline riscv-plic.adoc]: It sounds better with "sections below" instead :-)

File: riscv-plic.adoc [hunk 2]
```diff
@@ -17,102 +18,291 @@ https://creativecommons.org/licenses/by/4.0/.
...
+	base + 0x201000: Priority threshold for context 1
+	base + 0x201004: Claim/complete for context 1
+	...
+	base + 0x3FFE000: Priority threshold for context 15871
+	base + 0x3FFE004: Claim/complete for context 15871
+	base + 0x3FFE008: Reserved
+	...	
+	base + 0x3FFFFFC: Reserved
+	
+Below sections describe the control register blocks of PLIC operation parameters.
```
Comment (Kevin-Andes) [inline riscv-plic.adoc]: Same thing here. "Sections below" sounds better :-)

File: riscv-plic.adoc [hunk 3]
```diff
@@ -17,102 +18,291 @@ https://creativecommons.org/licenses/by/4.0/.
...
 
 ## Interrupt Priorities
 
-Each PLIC interrupt source can be assigned a priority by writing to its 32-bit
-memory-mapped `priority` register.  A priority value of 0 is reserved to mean
-``never interrupt'' and effectively dissables the interrupt. Priority 1 is the
-lowest active priority.  Ties between global interrupts of the same priority
-are broken by the Interrupt ID; interrupts with the lowest ID have the highest
-effective priority.
+If PLIC support Interrupt Priorities, then each PLIC interrupt source can be assigned a priority by writing to its 32-bit
```
Comment (Kevin-Andes) [inline riscv-plic.adoc]: One minor grammar suggestion:
   If PLIC supports interrupt priorities, ...

**Comment by changab** [comment]
> How about to revise it like this, "The PLIC multiplexes various device interrupts onto the external interrupt lines of hart contexts..."?
> I am not an English native person, maybe Palmer could make this better, this sentence is from him.
> Palmer?
Issue addressed

File: riscv-plic.adoc (hunk 1 above)
Comment (changab) [reply riscv-plic.adoc]: issue addressed

File: riscv-plic.adoc (hunk 2 above)
Comment (changab) [reply riscv-plic.adoc]: Issue addressed

File: riscv-plic.adoc (hunk 3 above)
Comment (changab) [reply riscv-plic.adoc]: Issue addressed

**Pull request merged**